dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -4,18 +4,33 @@
 import time
 import torch
 import statistics
+import argparse
 
+from fsspecfsdp import FSSpecFSDPStrategy
 from lightning import Trainer
+from lightning.pytorch.strategies import FSDPStrategy
 from lightning.pytorch.callbacks import ModelCheckpoint
 from lightning.pytorch.demos import (WikiText2)
 import torch.distributed
 from torch.utils.data import DataLoader
 
+DF_FSDP_STRATEGY = "dataflux_fsdp"
+FSDP_STRATEGY = "fsdp"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--strategy',
+                        choices=[DF_FSDP_STRATEGY, FSDP_STRATEGY],
```

Comment:
Did you try passing in non default option in GKE yaml file ? 

I thought we needed some work to support parser.args in GKE and hence demo has flags and GKE version has environment variables.

Reply:
I have not! But the image segmentation workload had no trouble parsing args from flags. It should work here too but I will double check.

Reply:
Made the dataflux fsdp strategy to be the default option. Re-ran the workload by passing `--strategy=fsspec_fsdp` and made sure it's running with no errors.

Reply:
Oh perfect. Thanks for testing it out!